content/dev/design-pattern-factory-method.md

```diff
@@ -0,0 +1,198 @@
+---
+type:           "post"
+title:          "Le Design Pattern 'Factory Method'"
```

Comment:
Yep, c'est du work in progress. Je suis parti d'une branche sur laquelle j'ai une dépendance et je n'avais pas encore changé la branche cible de ma MR.